packages/core/src/components/axes/axis.ts

```diff
@@ -186,20 +193,24 @@ export class Axis extends Component {
 				if (!scale.ticks(numberOfTicks).length) {
 					axis.tickValues([]);
 				} else {
-					const tickValues = scale
-						.nice(numberOfTicks)
-						.ticks(numberOfTicks);
+					const addSpaceOnEdges = Tools.getProperty(
+						options,
+						"timeScale",
+						"addSpaceOnEdges"
+					);
+
+					let tickValues;
+					if (addSpaceOnEdges) {
+						tickValues = scale
+							.nice(numberOfTicks)
+							.ticks(numberOfTicks);
+					} else {
+						tickValues = scale.ticks(numberOfTicks);
+					}
```

Comment:
Both the if and else blocks have `scale.ticks(numberOfTicks);`. Is it somehow possible to bring that portion outside so we can get rid of the else block?

Reply:
👍